app/router.js

```diff
@@ -7,25 +7,19 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('admin', {
-    resetNamespace: true
-  }, function() {
+  this.route('admin', function() {
     this.route('address');
     this.route('loaddb');
     this.route('lookup', { path: '/' });
-    this.route('users', {
-      resetNamespace: true
-    }, function() {
+    this.route('users', function() {
```

Comment:
This change is causing tests to fail because users sits at `/app/users`, not `/app/admin/users`